### PR TITLE
Merge toolbox to test-infra

### PR DESCRIPTION
**Pull request by yutongz**
https://github.com/istio/toolbox/issues/2

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -232,3 +226,13 @@ new_http_archive(
     url = "https://codeload.github.com/tianon/docker-brew-ubuntu-core/zip/b6f1fe19228e5b6b7aed98dcba02f18088282f90",
 )
 
+
+##
+## Testing
```
Comment (lookuptable) [inline WORKSPACE]: Can you put some details here?

**Comment by yutongz** [comment]
PTAL

File: WORKSPACE [hunk 2]
```diff
@@ -232,3 +226,13 @@ new_http_archive(
     url = "https://codeload.github.com/tianon/docker-brew-ubuntu-core/zip/b6f1fe19228e5b6b7aed98dcba02f18088282f90",
 )
 
+
+##
+## Repo(s) containing utilities for testing
```
Comment (lookuptable) [inline WORKSPACE]: Just `Repo` please.

**Comment by yutongz** [comment]
PTAL

**Comment by lookuptable** [comment]
LGTM

**Pull request merged**